code/Extensions/PageSlicesExtension.php

```diff
@@ -59,16 +58,11 @@ public function addSlicesCmsTab(FieldList $fields, $tabName = 'Root.Slices', $da
         );
 
 //        $gridConfig->addComponent(new GridFieldDataObjectPreview($this->previewer));
-//        $gridConfig->addComponent(new GridFieldVersionedOrderableRows('Sort'));
+        $gridConfig->addComponent(new GridFieldOrderableRows('Sort'));
```

Comment:
What's the reason behind this change? Slice is still a versioned data object, so this will break the sorting if you're looking at the CMS in Published mode I think.

Reply:
I should have remove this pull request when I understood it was tightly coupled with some code from Caltex. I will dig into it to understand everything, add a level of abstraction I guess, then rebase these commits